ext/standard/array.c

```diff
@@ -3776,10 +3776,16 @@ static zend_always_inline void php_array_merge_wrapper(INTERNAL_FUNCTION_PARAMET
 	HashTable *src, *dest;
 	uint32_t count = 0;
 
-	ZEND_PARSE_PARAMETERS_START(1, -1)
+	ZEND_PARSE_PARAMETERS_START(0, -1)
 		Z_PARAM_VARIADIC('+', args, argc)
 	ZEND_PARSE_PARAMETERS_END();
 
+	if (argc == 0) {
+		/* Return an empty array */
+		array_init_size(return_value, 0);
+		return;
```

Comment:
RETURN_EMPTY_ARRAY();

Reply:
There's a macro for everything... :) Adjusted.